framework_tvb/tvb/adapters/exporters/abcexporter.py

```diff
@@ -145,6 +154,53 @@ def is_data_a_group(self, data):
         """
         return isinstance(data, DataTypeGroup)
 
+    def gather_datatypes_for_copy(self, data, dt_path_list):
+        data_path = h5.path_for_stored_index(data)
+
+        if data_path not in dt_path_list:
+            dt_path_list.append(data_path)
+
+        with H5File.from_file(data_path) as f:
+            sub_dt_refs = f.gather_references()
+
+            for _, ref_gid in sub_dt_refs:
+                if ref_gid:
+                    dt = load.load_entity_by_gid(ref_gid)
+                    self.gather_datatypes_for_copy(dt, dt_path_list)
+
+    def prepare_datatypes_for_export(self, data):
+        all_datatypes = self._get_all_data_types_arr(data)
+
+        # We are exporting a group of datatype measures so we need to find the group of time series
+        if hasattr(all_datatypes[0], 'fk_source_gid'):
+            ts = h5.load_entity_by_gid(all_datatypes[0].fk_source_gid)
+            data_2 = dao.get_datatypegroup_by_op_group_id(ts.parent_operation.fk_operation_group)
+            all_datatypes_2 = self._get_all_data_types_arr(data_2)
+            all_datatypes = all_datatypes_2 + all_datatypes
+        else:
+            data_2 = dao.get_datatype_measure_group_from_ts_from_pse(all_datatypes[0].gid, DatatypeMeasureIndex)
+            all_datatypes_2 = self._get_all_data_types_arr(data_2)
+            all_datatypes = all_datatypes + all_datatypes_2
+
+        if all_datatypes is None or len(all_datatypes) == 0:
+            raise ExportException("Could not export a data type group with no data!")
+
+        op_file_dict = dict()
+        for dt in all_datatypes:
+            h5_path = h5.path_for_stored_index(dt)
+            StorageInterface().get_storage_manager(h5_path).remove_metadata('parent_burst', check_existence=True)
```

Comment:
You are totally right this one, it's weird that it doesn't generate any error so that's why I didn't see it. I am moving the deletion of parent_burst after the files have been copied.